cfgov/jinja2/v1/_includes/molecules/hero.html

```diff
@@ -91,7 +82,7 @@ <h1 class="m-hero_heading">{{ value.heading | safe }}</h1>
             filter: progid:DXImageTransform.Microsoft.AlphaImageLoader(
                 src='{{ sm_img.url }}',
                 sizingMethod='scale');
-            padding-bottom: {{ bp_xs_padding }};
+            padding-bottom: {{ ( sm_img.height / sm_img.width * 100 ) }}%;
```

Comment:
Are the parentheses doing anything here? Seems like they should be around the division or the multiplication.